src/pkg/lint/validate_schema_test.go

```diff
@@ -0,0 +1,71 @@
+package lint
+
+import (
+	"os"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/types"
+	goyaml "github.com/goccy/go-yaml"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateSchema(t *testing.T) {
+	readSchema := func(t *testing.T) []byte {
+		t.Helper()
+		zarfSchema, err := os.ReadFile("../../../zarf.schema.json")
+		if err != nil {
+			t.Fatalf("Error reading schema file: %s", err)
+		}
+		return zarfSchema
+	}
+
+	readAndUnmarshalYaml := func(t *testing.T, path string) interface{} {
+		t.Helper()
+		var unmarshalledYaml interface{}
+		file, _ := os.ReadFile(path)
+		goyaml.Unmarshal(file, &unmarshalledYaml)
+		return unmarshalledYaml
+	}
+
+	readAndUnmarshallZarf := func(t *testing.T, path string) types.ZarfPackage {
```

Comment:
```suggestion
	readAndUnmarshallZarfPackage := func(t *testing.T, path string) types.ZarfPackage {
```

Likely better to refer to this as a Package (or Pkg if you prefer) since Zarf can be many things.